Corona-Warn-App/src/main/res/layout/include_main_overview_row.xml

```diff
@@ -43,7 +43,7 @@
                 android:layout_height="@dimen/icon_size_main_card"
                 android:importantForAccessibility="no"
                 android:src="@{icon}"
-                android:tint="@{FormatterHelper.formatColorIcon(iconTint)}"
+                android:tint="@{(iconTint != 0) ? iconTint : @color/colorAccentTintIcon}"
```

Comment:
Maybe keep the method `formatColor` and provide the fallback color resource as a parameter. 
and can be used in here and  include_divider.xml

Reply:
method is buggy. better not.